### PR TITLE
Support all float dtypes in `F.gaussian`

### PR DESCRIPTION
**Pull request by takagi**
This PR follows #4582 to support all float dtypes in `F.gaussian`.

### CODE REVIEW

**Comment by takagi** [comment]
Jenkins, test this please.

File: foo [hunk 1]
```diff
@@ -0,0 +1,302 @@
+============================= test session starts ==============================
+platform linux -- Python 3.6.3, pytest-3.3.1, py-1.5.2, pluggy-0.6.0
```
Comment (okuta) [inline foo]: Please remove this file.

**Comment by okuta** [comment]
jenkins, test this please.

**Comment by okuta** [comment]
LGTM!

**Pull request merged**